packaging/src/docker/README.md

```diff
@@ -210,3 +210,63 @@ docker compose down
     select count(distinct a) from hive_example;
     select sum(b) from hive_example;
   ```
+
+#### `sys` Schema and `information_schema` Schema
+
+`Hive Schema Tool` is located in the Docker Image at `/opt/hive/bin/schematool`.
+
+By default, system schemas such as `information_schema` for HiveServer2 are not created. 
+To create system schemas for a HiveServer2 instance, 
+users need to configure HiveServer2 to use a remote Hive Metastore Server and use a database other than embedded Derby for the Hive Metastore Server.
+
+Assuming `Maven` and `Docker CE` are installed, a possible use case is as follows.
+Create a `compose.yaml` file in the current directory,
+
+```yaml
+services:
+  some-postgres:
+    image: postgres:17.2-bookworm
+    environment:
+      POSTGRES_PASSWORD: "example"
+  metastore-standalone:
+    image: apache/hive:4.0.1
+    depends_on:
+      - some-postgres
+    environment:
+      SERVICE_NAME: metastore
+      DB_DRIVER: postgres
+      SERVICE_OPTS: >-
+        -Djavax.jdo.option.ConnectionDriverName=org.postgresql.Driver
+        -Djavax.jdo.option.ConnectionURL=jdbc:postgresql://some-postgres:5432/postgres
+        -Djavax.jdo.option.ConnectionUserName=postgres
+        -Djavax.jdo.option.ConnectionPassword=example
+    volumes:
+      - ~/.m2/repository/org/postgresql/postgresql/42.7.5/postgresql-42.7.5.jar:/opt/hive/lib/postgres.jar
+      - ~/.m2/repository/org/checkerframework/checker-qual/3.48.3/checker-qual-3.48.3.jar:/opt/hive/lib/checker-qual.jar
+  hiveserver2-standalone:
+    image: apache/hive:4.0.1
+    depends_on:
+      - metastore-standalone
+    environment:
+      SERVICE_NAME: hiveserver2
+      IS_RESUME: true
+      SERVICE_OPTS: >-
+        -Djavax.jdo.option.ConnectionDriverName=org.postgresql.Driver
+        -Djavax.jdo.option.ConnectionURL=jdbc:postgresql://some-postgres:5432/postgres
+        -Djavax.jdo.option.ConnectionUserName=postgres
+        -Djavax.jdo.option.ConnectionPassword=example
+        -Dhive.metastore.uris=thrift://metastore-standalone:9083
+    volumes:
+      - ~/.m2/repository/org/postgresql/postgresql/42.7.5/postgresql-42.7.5.jar:/opt/hive/lib/postgres.jar
+      - ~/.m2/repository/org/checkerframework/checker-qual/3.48.3/checker-qual-3.48.3.jar:/opt/hive/lib/checker-qual.jar
```

Comment:
nit: what's the purpose of `checker-qual-3.48.3.jar`, I'm not sure if HS2 or HMS uses it.

Reply:
- `org.checkerframework:checker-qual:3.48.3` is a Runtime Dependency of `org.postgresql:postgresql:42.7.5`. See https://mvnrepository.com/artifact/org.postgresql/postgresql/42.7.5 . Should I remove `org.checkerframework:checker-qual:3.48.3`? 
- It looks like https://github.com/pgjdbc/pgjdbc/pull/2262 has been in dispute for nearly 3 years.

Reply:
In this example, I can query the table without this dependency, so let's just remove it

Reply:
- Done. Just waiting for CI of current PR with https://github.com/apache/hive-site/pull/40 .